src/liger_kernel/transformers/monkey_patch.py

```diff
@@ -268,6 +267,8 @@ def apply_liger_kernel_to_phi3(
     if cross_entropy:
         modeling_phi3.CrossEntropyLoss = LigerCrossEntropyLoss
     if fused_linear_cross_entropy:
+        from liger_kernel.transformers.model.phi3 import lce_forward as phi3_lce_forward
```

Comment:
maybe we can move this back. we don't want to support too old transformers

Reply:
oh! apparently i swapped the `0` and `1`, lemme revert these phi3 related stuff